raft-tech/TANF-app
# 3025 As an STT user, I need an accurate error report when I space-fill `COUNTY_FIPS_CODE`

## Summary of Changes

Pull request closes #3025. 
This PR fixes the issue for when report is space filled for COUNTY_FIPS_CODE. The Since the field is space filled it is considered as None in which case throws error for applying ```lstrip``` function to None value. 

## How to Test

Run the following commands to start the 
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
```

1. Open http://localhost:3000/ and sign in.
1. Proceed with functional tests as described herein.
1. Submit test file that has space filled COUNTY_FIPS_CODE (see test file in ticket)
1. Make sure the error is clear for not providing value for COUNTY_FIPS_CODE

## Deliverables
_More details on how deliverables herein are assessed included [here](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:
- [x] space-filled `COUNTY_FIPS_CODE` raises cat 2 error message in error report. 
- [x] T1/M1/T2/M2 record associated with space-filled `COUNTY_FIPS_CODE`  is still parsed
- [x] cat 4 error message are not raised due to the space-filled `COUNTY_FIPS_CODE` 

### [Deliverable 2: Tested Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](https://github.com/raft-tech/TANF-app/blob/develop/docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
